docker/run_jakartaeetck.sh

```diff
@@ -256,6 +249,15 @@ if [[ $test_suite == ejb30/lite* ]] || [[ "ejb30" == $test_suite ]] ; then
   sed -i.bak 's/-Xmx1024m/-Xmx4096m/g' ${TS_HOME}/bin/ts.jte
```

Comment:
@starksm64 will the ^ sed command on line 249 work on your machine?

Reply:
If changes are needed, let me know and I'll create a new pr.  Merging since this is approved.